### ISSUE TITLE
Headers not reaching in step 46 of assembly

### ISSUE THREAD

**Issue by ctag** [makersmakingchange/LipSync-Classic#4]
Hi,

I'm at the tail end of assembling a lipsync, and have hit a small roadblock. Step 46 in the manual shows joining the joystick input onto the main PCB. When I try to do this, the headers just *[barely](https://lh3.googleusercontent.com/-HzDvTFKiTPs4Q3U8kpLXTWcPFMRZtS6OtpZfv3VrYSGvlZRsmPxdE6zdAhsjINIq8nM0WegqKW7vgM3H8OEQwK9bu-DROiDsiqMQHMnjDnIxqmINXWQZl38aM0Gi0g8xnji6cWNvCtDMhDniQn_GvPznPVhrXtDz6Lk0yiw5tF2zVp4Xqr8-Ndb2lhKOlgLu283Etzqd2gsMgY9rRNgOvsjvHW8KYzZf_m3EBcSYTXbwPBonm5OGFSADbklnt6JssEMPtlCppNgmtlF1SCmGRc3a79JIT2K9ro4Ktno046k104-RLC7nrI0yqNmCLkPhakF1mqQCSRqFbXuISUqnmZ-qAARwxpwyUrDOy8qILJqgdyt1xg64l7FCB7AegzQ-UdPYlkpJ51PcGy9UY0Od8dYBS05BZNpyt0ALCffYN-3-FkpJWQePbKxkZeqrGjp3NsC86aRNtH3U7XGeZQ6GRtzNLt-WrFsuX61oYIV6rgaKQyuOEwF5rj4aCnPcuy_crPsSqgiXqwjjBdL-_Z0DPpU464crkgGK9ZTdSAfJGMwY9TNBFeKbMZm-w07cDzgIPg_hLTLVkgbFrQhIPji7mB7zcZy9ECg0ERDlsRe26Q1Vu7eGPZqD0Zlpx-I7A0ok3Ibmz9gXKxL43OZXadcXSOUUC3xIztQ=w3201-h1602-no)* reach, and I doubt they will stay joined through prolonged use. Not unexpected, since the right angle headers are just from a drawer in my office. 

My first guess to address this is to just solder the headers together, if the system needs to be serviced later they'll have to be removed and possibly replaced. **But** in Step 47 there's.. something else there. I can't tell from the picture, but it looks like the headers have been boosted another mm by something. Is there any documentation you can give me about that, or am I looking at the image wrong?

Thanks!

**Comment by hpigot** [comment]
Hey ctag, I noticed the same issue and had the same thoughts (using parts directly from MakersMakingChange/NSS). It seems like the part (female headers) changed, but the assembly manual images weren't brought up to date. They do make a special comment about ONLY using those female headers from DigiKey, but I'm not sure what motivated the change.
![image](https://user-images.githubusercontent.com/22122190/43164889-b417e34e-8f92-11e8-9502-08843ce0328e.png)

The device did work as-is, but I cannot comment on the long-term functionality. Perhaps @igourlay can comment on this?

**Comment by milador** [comment]
Hi ctag and hpigot, I am one of the engineers from MakersMakingChange/NSS.

Thank you for bringing up the issue. The images in step 47 are from older version of LipSync PCBs (Version 2.5). We used right-angle female headers and straight male headers to connect main board to joystick board in version 2.5.

We switched to right-angle male headers and straight female headers instead in version 2.6.
You would need to use the headers specified in the list of components as not all the common headers would work due to their height difference.

I hope it helps and we will update the pictures in step 47 in the next update.

**Comment by hpigot** [comment]
Thanks for the input milador. Perhaps in 2.7 the connection between the headers could be made more deeply; at the moment the male headers barely touch the female headers and it does seem out of place during construction.

**Comment by milador** [comment]
@hpigot Thank you for your feedback.We will look into it and update you.

**Comment by ctag** [comment]
Thank you for the responses. I'm glad I wasn't going crazy looking at those pictures.

As hpigot wrote, the headers just barely reach. I did use the vertical female headers linked to in the manual (part #12) but used generic right angle male headers (part #11).

Since I plan to hold on to this lipsync as a reference when making future ones, I went ahead and [soldered the joint for now](https://lh3.googleusercontent.com/3P8OnEv3PJSzcNVY25qphE0EQzSgjAcXkquns4NXm3_3wWt2QPPuaNxUH0YX04YDXlKZLpmQgE_wDAF9Sjag_rvHZCmwtjYCX4f0OsWeobROQn48OZpCxLdjUqaUag6aRAb_4zmW99Zk18i-UkyIsl5VShYhL0fnRuKXyPJ2_XSTfJ2oFSnYa7tQRNrydXnjw_Fp3ncasW3TB6PBrvYEsl7zXH8CiDXzQDYd5fiwYldPvZdjcg4HfzEg2e11ZxucvWHDAEl4qyvBjkN3MmXI0PNDnV_1K-VKnZd3IfAn5oGOfpLH5O_XDUVMm3ElKziQg6LD5Fbk7ZSAAGR5zSdem2m_XFYFWFmfDH9rgjotMkvNW8DHSgaBxJUNGfKsr5V2g8ZZAr6kpix1XydUTWe8Fz-iyN3nAU9irgP2V568oFqn5oLjWRulvTfwdFRy7LgTGdPwM13OswgsWuKKSlSaqBsl4ixSr7p-tXpL3IHLGdhn7tlKWUeu5_DeB8wUY290ElJE5iK8mDNNHA4nmg15M8D_qWj3O19b-Iez7H1wdck-4YYTHaUdhZFcPce84eJ12CRHo46zAHiEyp8Am7tKdqBWHzJxv6zYIUlQNy4=w3201-h1602-no), which seems to work nicely.
